checks/raw/shell_download_validate.go

```diff
@@ -977,7 +961,8 @@ func collectFetchProcSubsExecute(startLine, endLine uint, node syntax.Node, cmd,
 				EndOffset: endLine,
 				Snippet:   cmd,
 			},
-			Type: checker.DependencyUseTypeDownloadThenRun,
+			Pinned: asBoolPointer(false),
```

Comment:
this one i'm not sure about. can we detect a pinned version of this one? 
probably fine as hard coded false?

Reply:
My understanding is that all current cases of `downloadThenRun` identified by Scorecard are considered unpinned.

Here's a related issue where Joyce comments that yum, wget and curl generate warnings but cannot be pinned.
https://github.com/ossf/scorecard/issues/2578

Reply:
I believe it's correct to hardcode as false.